packages/lesswrong/components/posts/PostsListSettings.tsx

```diff
@@ -66,6 +66,10 @@ const FILTERS_ALL = {
       label: "Questions",
       tooltip: "Open questions and answers, ranging from newcomer questions to important unsolved scientific problems."
     },
+    events: {
+      label: "Events",
+      tooltip: "Events from around the world."
+    },
```

Comment:
Copied from the LW list above - not sure how useful this view is, but might be handy for testing at least